src/Common/Description/GenericDescription.tsx

```diff
@@ -1,327 +1,238 @@
 
-// Import images from dashboard
-// import { ReactComponent as HeaderIcon } from '../../../assets/icons/mdeditor/ic-header.svg'
-// import { ReactComponent as BoldIcon } from '../../../assets/icons/mdeditor/ic-bold.svg'
-// import { ReactComponent as ItalicIcon } from '../../../assets/icons/mdeditor/ic-italic.svg'
-// import { ReactComponent as LinkIcon } from '../../../assets/icons/mdeditor/ic-link.svg'
-// import { ReactComponent as StrikethroughIcon } from '../../../assets/icons/mdeditor/ic-strikethrough.svg'
-// import { ReactComponent as CodeIcon } from '../../../assets/icons/mdeditor/ic-code.svg'
-// import { ReactComponent as QuoteIcon } from '../../../assets/icons/mdeditor/ic-quote.svg'
-// import { ReactComponent as ImageIcon } from '../../../assets/icons/mdeditor/ic-image.svg'
-// import { ReactComponent as OrderedListIcon } from '../../../assets/icons/mdeditor/ic-ordered-list.svg'
-// import { ReactComponent as UnorderedListIcon } from '../../../assets/icons/mdeditor/ic-unordered-list.svg'
-// import { ReactComponent as CheckedListIcon } from '../../../assets/icons/mdeditor/ic-checked-list.svg'
+import Moment12HourFormat from './constant'
 
 const GenericDescription = ({
-    isClusterTerminal,
-    clusterId,
-    isSuperAdmin,
-    appId,
-    descriptionId,
-    initialDescriptionText,
-    initialDescriptionUpdatedBy,
-    initialDescriptionUpdatedOn,
+    descriptionText,
+    descriptionUpdatedBy,
+    descriptionUpdatedOn,
     initialEditDescriptionView,
-    updateCreateAppFormDescription,
-    appMetaInfo,
     tabIndex,
+    updateDescription,
+    title,
 }: GenericDescriptionProps) => {
     const [isEditDescriptionView, setEditDescriptionView] = useState<boolean>(initialEditDescriptionView)
-    const [descriptionText, setDescriptionText] = useState<string>(initialDescriptionText)
-    const [descriptionUpdatedBy, setDescriptionUpdatedBy] = useState<string>(initialDescriptionUpdatedBy)
-    const [descriptionUpdatedOn, setDescriptionUpdatedOn] = useState<string>(initialDescriptionUpdatedOn)
-    const [modifiedDescriptionText, setModifiedDescriptionText] = useState<string>(initialDescriptionText)
+    const [modifiedDescriptionText, setModifiedDescriptionText] = useState<string>(descriptionText)
     const [selectedTab, setSelectedTab] = useState<MDEditorSelectedTabType>(MD_EDITOR_TAB.WRITE)
     const isDescriptionModified: boolean = !deepEqual(descriptionText, modifiedDescriptionText)
     const mdeRef = useRef(null)
 
-    useEffect(() => {
-        if (typeof updateCreateAppFormDescription === 'function') {
-            updateCreateAppFormDescription(modifiedDescriptionText)
-        }
-    }, [modifiedDescriptionText])
-
-    useEffect(() => {
-        if (!isClusterTerminal && appId === 0) {
-            mdeRef.current?.finalRefs?.textarea?.current?.focus()
-        }
-    }, [isEditDescriptionView])
+    const _moment = moment(descriptionUpdatedOn, 'YYYY-MM-DDTHH:mm:ssZ')
```

Comment:
please use dayjs and use constant for timezone

Reply:
will do later, dayjs not available in common-lib